doc/release-notes.md

```diff
@@ -65,6 +65,14 @@ Notable changes
 P2P and network changes
 -----------------------
 
+- The mempool now tracks transactions submitted via the wallet or rpcs as
+  unbroadcast transactions. Every 10-15 minutes, the node will try to announce
+  unbroadcast transactions until a peer requests it via a `getdata` message or
+  the transaction is removed from the mempool for other reasons. If you have a
+  custom setup to use P2P messages to submit a wallet transaction to your node
+  while it is not connected to other nodes, these changes reduce your initial
+  broadcast guarantees. (#18038)
+
 Updated RPCs
 ------------
 
```

Comment:
Since it seems you'll need to include the rpc info [from #18895](https://github.com/bitcoin/bitcoin/pull/18895#issuecomment-630998986) I wanted to help draft a description:
```suggestion
- `getmempoolinfo` now returns an additional `unbroadcastcount` field which represents the number of transactions that the mempool considers unbroadcast.
- Mempool RPCs such as `getmempoolentry` and `getrawmempool` with `verbose=true`now return an additional "unbroadcast" field which indicates whether the mempool considers this transaction unbroadcast.
```

Reply:
thanks! added with some extra info